### PR TITLE
[ADD] estate: basic directory structure T#69469

### PR DESCRIPTION
**Pull request by Suavesito-Olimpiada**
@deivislaya 

This PR is ready for review. It includes the basic keys in `__manifest__.py` so that `pre-commit-vauxoo` pass (both mandatory and optional checks) and the technical documentation is respected.

Regards!

### CODE REVIEW

**Comment by deivislaya** [comment]
Why a new MR with the same changes?

File: estate/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'Real Estate',
```
Comment (deivislaya) [inline estate/__manifest__.py:4]:
```suggestion
    'name': 'Real Estate',
    version': '16.0.1.0.0',
```

**Review by deivislaya — CHANGES_REQUESTED** [review]

**Comment by Suavesito-Olimpiada** [comment]
I changed the root commit to add to the `.gitignore` file the names of the files created by `pre-commit-vauxoo` tool, this caused the previous PR to be automatically closed.

File: estate/__manifest__.py (hunk 1 above)
Comment (Suavesito-Olimpiada) [reply estate/__manifest__.py:4]: I'll add the version, it was an oops. 😅

**Comment by Suavesito-Olimpiada** [comment]
@deivislaya 

Comments and corrections were addressed, this PR is ready to merge (I hope).

Regards!

**Pull request merged**